bus/bus.go

```diff
@@ -59,7 +59,7 @@ type (
 	ContractStore interface {
 		Contracts() ([]rhpv2.Contract, error)
 		Contract(id types.FileContractID) (rhpv2.Contract, error)
-		AddContract(c rhpv2.Contract) error
+		AddContract(hk consensus.PublicKey, c rhpv2.Contract) error
```

Comment:
you can get the host key from the `rhpv2.Contract`

Reply:
I don't think so?

Reply:
it has a `.HostKey()` method!